docs/content/commands/manpages/store/geowave-addstore.txt

```diff
@@ -80,26 +80,37 @@ When the `redis` type option is used, additional options are:
 
 *--compression* _<compression>_::
   The compression to use.  Possible values are `snappy`, `lz4`, and `none`. Default is `snappy`.
- 
+
+*--serialization* _<serialization>_::
+  Can be \"fst\" or \"jdk\". Defaults to fst. This serialization codec is only used for the data index when secondary indexing.
+
 When the `rocksdb` type option is used, additional options are:
 
 *--dir* _<path>_::
   The directory to read/write to.  Defaults to "rocksdb" in the working directory.
-   
+
 *--compactOnWrite* _<enabled>_::
   Whether to compact on every write, if `false` it will only compact on merge. Default is `true`.
 
 *--batchWriteSize* _<count>_::
   The size (in records) for each batched write. Anything <= 1 will use synchronous single record writes without batching. Default is 1000.
 
+When the `filesystem` type option is used, additional options are:
+
+*--dir* _<path>_::
+  The directory to read/write to.  Defaults to "geowave" in the working directory.
+
+*--format* _<format>_::
+  Optionally use a formatter configured with Java SPI of type org.locationtech.geowave.datastore.filesystem.FileSystemDataFormatterSpi.  Defaults to 'binary' which is a compact geowave serialization.  Use `geowave util filesystem listformats` to see available formats.
```

Comment:
yeah, I get that and I can leave that out. But this is an odd one because the only "option" is "binary" without going the SPI route so it is pretty much a non-option without some explanation about how to get additional options through SPI.